upgrade_test.py

```diff
@@ -501,6 +504,11 @@ def shuffle_nodes_and_alternate_dcs(nodes: List[BaseNode]) -> List[BaseNode]:
 
         return [x for x in chain.from_iterable(zip_longest(*dc_nodes.values())) if x]
 
+    @staticmethod
+    def _update_scylla_yaml_on_node(node_to_update: BaseNode, updates: dict):
+        with ScyllaYamlUpdateEvent(node=node_to_update.name), node_to_update.remote_scylla_yaml() as scylla_yaml:
```

Comment:
I can understand why we'll want an event
but i'm not sure i'm following why it's need to be `ScyllaDatabaseContinuousEvent`, i.e. do we cat about it's start and end time ?